test/feature/redirection.feature

```diff
@@ -0,0 +1,22 @@
+Feature: reverse routing
+
+  Scenario: standard redirection
+    Given the routing definition
+      """
+      GET '/some/route', to: redirect('/redirected/route')
+      """
+    And an exprestive app using defaults
```

Comment:
I think this line should either move to a `Background` block, or it should be implicit.